dblp-aliases.csv

```diff
@@ -3532,7 +3532,7 @@ Aluysius Maria Ardi Handojoseno,Ardi Handojoseno
 Álvar-Gines Legaz-Aparicio,Álvar Legaz-Aparicio
 Alvaro A. A. De Salles,Álvaro Augusto Almeida de Salles
 Alvaro A. B. Lourenço,Alvaro Lourenço
-Alvaro A. Cárdenas-Mora,Alvaro A. Cárdenas
+Alvaro A. Cárdenas-Mora,Alvaro A. Cárdenas,Alvaro Cardenas
```

Comment:
Please remove this - the `dblp-aliases.csv` file is now generated automatically.